### DISCUSSION TITLE
Updating Mosquitto LXC

### DISCUSSION THREAD

**Discussion by borpin** [tteck/Proxmox#1650] [Help Needed]
I am trying to update my Mosquitto LXC created by the script a few months ago.

I get the following error
```
E: The repository 'https://repo.mosquitto.org/debian bullseye Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://security.debian.org bullseye-security Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ftp.debian.org/debian bullseye Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'http://ftp.debian.org/debian bullseye-updates Release' no longer has a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```
I know there have been some changes to mosquitto recently and I suspect this is the issue.

Is there a simple way to update the LXC or should I start from scratch?

**Comment by tteck** [comment]
Since Mosquitto doesn't receive frequent updates, my suggestion would be to temporarily comment out the repository in `/etc/apt/sources.list.d/` for now if you upgraded to Debian 12

**Reply by tteck** [reply]
The new installation uses the distribution package.

**Comment by borpin** [comment]
Very odd.

First, noticed that it wasn't just the mosquitto repos, actually, cannot contact any repo.

```
root@mqtt:~# ping security.debian.org
ping: connect: Network is unreachable
```
Any suggestions? I can contact machines on the local network.

I have a Pi-Hole LXC I created from a script at the same time and that is fine. As far as I am aware, I haven't done anything to this that might cause this.

I cannot install `net-tools` either of course to investigate.

Looks like I'll have to create a new one anyway.

**Reply by tteck** [reply]
DNS

**Reply by tteck** [reply]
I thought that you were getting the error because you upgraded MQTT to Debian 12.

**Reply by borpin** [reply]
No, I've done nothing to the LXC.

`nameserver` entry is correct and I can ping it.

This all started because I cannot connect to the broker from another subnet!

**Reply by tteck** [reply]
It appears that you have caused some kind of network issue.

**Reply by borpin** [reply]
Yes and I cannot figure out what.

I'm just going to create a new instance. Cheers
